frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1280,6 +1280,28 @@ const ManageHostsPage = ({
   ) => {
     evt.preventDefault();
 
+    const hiddenColumnsStorage = localStorage.getItem("hostHiddenColumns");
+    let currentHiddenColumns;
+    let visibleColumns;
+    if (hiddenColumnsStorage) {
+      currentHiddenColumns = JSON.parse(hiddenColumnsStorage);
+    }
+
+    if (config && currentUser) {
+      const tableColumns = generateVisibleTableColumns(
+        currentHiddenColumns,
+        config,
+        currentUser,
+        currentTeam
+      );
+
+      const columnAccessors = tableColumns.map((column) =>
+        column.accessor ? column.accessor : ""
+      );
+      columnAccessors.shift();
```

Comment:
@lukeheath, is the `shift()` here to remove the checkbox column or is it something else?  If it is intended to pull out the selection column, does it work as expected for users who don't have access to this column. There's some complex logic around that in the `generateTableHeaders` function in the `HostsTableConfig` and I'm not sure if it lines up with the assumptions here.

Reply:
@gillespi314 Ack! Great catch. This was intended to remove the checkbox, and I forgot that some roles don't have access to the checkbox. I'll fix this real quick.

Reply:
Updated! I'm now removing all empty strings, so this should only apply if the checkbox is present.

Reply:
Looks good! I noticed a `console.log` so if you want to remove it before merging if can reapprove.